README.md

```diff
@@ -1,8 +1,13 @@
+[![Cosmos ecosystem][cosmos-shield]](https://cosmos.network)
+
+# ICS 23
+
 [![Go](https://github.com/cosmos/ics23/actions/workflows/go.yml/badge.svg?branch=master)](https://github.com/cosmos/ics23/actions/workflows/go.yml)
+[![Go coverage](https://codecov.io/github/cosmos/ics23/branch/master/graph/badge.svg?token=xlGriS907o&flag=go)](https://codecov.io/gh/cosmos/ics23/tree/master/go)
 [![Rust](https://github.com/cosmos/ics23/actions/workflows/rust.yml/badge.svg?branch=master)](https://github.com/cosmos/ics23/actions/workflows/rust.yml)
+[![Rust coverage](https://codecov.io/github/cosmos/ics23/branch/master/graph/badge.svg?token=xlGriS907o&flag=rust)](https://codecov.io/gh/cosmos/ics23/tree/master/rust)
 [![TypeScript](https://github.com/cosmos/ics23/actions/workflows/typescript.yml/badge.svg?branch=master)](https://github.com/cosmos/ics23/actions/workflows/typescript.yml)
```

Comment:
should we separate this into a table of languages? easier to see what's what

Reply:
Done